src/src/common.c

```diff
@@ -99,14 +95,13 @@ void setPowerdB(float dB)
     } else
     {
       rtc6705PowerAmpOn();
-      target_set_power_dB(dB);
```

Comment:
Very minor `dB` variable here for a little speed optim.

Reply:
Ok, I'll revert it.